x-pack/plugins/elastic_assistant/server/routes/user_conversations/create_route.ts

```diff
@@ -41,27 +41,26 @@ export const createConversationRoute = (router: ElasticAssistantPluginRouter): v
         const assistantResponse = buildResponse(response);
         try {
           const ctx = await context.resolve(['core', 'elasticAssistant', 'licensing']);
-          const license = ctx.licensing.license;
-          if (!hasAIAssistantLicense(license)) {
-            return response.forbidden({
-              body: {
-                message: UPGRADE_LICENSE_MESSAGE,
-              },
-            });
+          // Perform license, authenticated user and FF checks
+          const checkResponse = performChecks({
+            authenticatedUser: true,
+            capability: 'assistantKnowledgeBaseByDefault',
```

Comment:
We'll want to remove `capability` as the conversations API isn't behind a feature flag:

```suggestion
          // Perform license and authenticated user checks
          const checkResponse = performChecks({
            authenticatedUser: true,
```